mozilla/fx-private-relay
# MPP-3443 & MPP-3444 - Free user onboarding experience

This PR fixes MPP-3443 & MPP-3444
  
# New feature description

Figma: https://www.figma.com/file/VWDfo6NG52RbGmWGfuKX2F/Improved-Onboarding-for-Free-Relay-Users?type=design&node-id=725-1791&mode=design&t=4MiMImZYht3nGuQz-0 

- Has all three steps - welcome to relay, copy your mask to use it, use your masks everywhere.
- Second step (copy your mask to use it) has additional functionality to try out email forwarding, which triggers sending email.
- Third step checks if extension is installed.
- Congratulatory animation is triggered when user finishes onboarding.
- Onboarding state is updated for user so they don’t go through onboarding again. 

Implement the additional functionality needed for the second step of the free user onboarding, where a user can copy their mask and see additional information about their mask. This is broken into its own task as it doesn’t block the overall flow of onboarding. 

Majority of UI has been reused from existing screens. This will require a few new strings, but overall, it will require a bit of QA testing. 

# Screenshot (if applicable)

 https://vimeo.com/871702930/e8c8afdf67?share=copy

# How to test

- Make sure user profile you are using has `onboarding_free_state` set to 0 - you can do this from the admin dashboard. 
- `free_user_onboarding` flag should be turned on as well. 
- Copy over the `MAX_FREE_ONBOARDING` env var value into your env. file
- Run `npm run watch`

# Checklist (Definition of Done)
